citi-onboarding/virtu-onepage
# Hotfix/navbar navigation

### What I did
- Made navbar navigate
- Added [ScrollSpy](https://www.npmjs.com/package/react-scrollspy) module to auxiliate on the development.
- Made it work responsively

### How to test
1. Enter server folder with `cd server`
2. Execute the command `yarn` to download the server modules
3. Execute the command `yarn start` to start the server
4. Execute the command `cd ../client` and then `yarn` to download the client modules
5. Execute the command `yarn start` to start the client (**IMPORTANT**, since I installed a new package)
6. Open your browser on `http://localhost:3001/admin` to view keystone admin
7. Log in with the credentials available at Notion
8. Open another browser page on `http://localhost:3000` to see the client page
9. Check if the navbar is going to the right section when you click it, and the text is getting bolder.
10. Check if when you scroll the page, the respective names of the sections on the navbar get bolder.

### OBS
- I'll probably need to pull the other navbar branch into this one to make the responsive part work correctly

Resolves #29
